app/models/oauth_token.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class OauthToken < ApplicationRecord
+  OAUTH_APPLICATION_NAME = 'ic-rails-huey-toby'
+  OAUTH_APPLICATION_REDIRECT_URI = 'urn:ietf:wg:oauth:2.0:oob'
+
+  class << self
+    # https://github.com/doorkeeper-gem/doorkeeper/issues/846#issuecomment-230297646
```

Comment:
Let's add a comment to avoid all devs to have to read this issue 🙈 

```suggestion
		# Generate token manually to automatically log a user in after registering    
		# https://github.com/doorkeeper-gem/doorkeeper/issues/846#issuecomment-230297646
```